docs/getting_started/installation.md

```diff
@@ -0,0 +1,55 @@
+# Installation
+
+BIDS App for combining multiple atlas parcellations
+
+## Requirements
+
+* Docker (Intel Mac/Windows/Linux) or Singularity (Linux)
+
+### Notes
+* Inputs to Labelmerge should be organized as BIDS derivatives, taking one "base" and one "overlay" atlas
```

Comment:
Should provide a link to the BIDS derivatives spec so that users can see an example.